backend/core/settings.py

```diff
@@ -36,7 +36,7 @@
 # Elastic Search
 # https://django-elasticsearch-dsl.readthedocs.io/en/latest/quickstart.html
 ELASTICSEARCH_DSL = {
-    "default": {"hosts": "localhost:9200"},
+    "default": {"hosts": "host.docker.internal:9200"},
```

Comment:
Why did you need to change this? Did localhost not work?

Reply:
Yes... localhost refers to only local django container. Since we are referring to es container, we need to indicate that as "host.docker.internal"

Reply:
Oh, I think I see what's wrong. The problem is that I've set up react, django and pgdb to be on the default bridge network and I've set elasticsearch and kibana to be on a separate bridge network. So django and elasticsearch can't communicate inside docker because they are on different networks.

`host.docker.internal` is for the container contacting the host... in this case I think you had to change it to this because django can't reach elasticsearch through the network inside docker so it has to access elasticsearch via the host

I need to fix this and put them all on the same network. But I guess you can do this for now and I when I fix it I will change this. I think the host should actually be  `elasticsearch:9200`

Reply:
I'm glad I asked because I wouldn't have realized this otherwise. Fix is in my PR.